openmdao/docs/features/recording/getting_values.rst

```diff
@@ -134,3 +134,23 @@ runs of the model.
 .. embed-code::
     openmdao.recorders.tests.test_sqlite_recorder.TestFeatureSqliteRecorder.test_feature_load_system_case_for_restart
     :layout: interleave
+
+Loading a DataBase into Memory
+------------------------------
+
+Every time the `get_case` method is used, the case reader is making a new query
+to the DB (with the exception of recurring requests, which are cached). This doesn't
```

Comment:
suggest replacing "DB" with "database" (i.e. not using shorthand in documentation)